### PR TITLE
Added openshift ci config for image-controller repo

### PR DESCRIPTION
**Pull request by tisutisu**
On boarding openshift ci to the image-controller repo

### CODE REVIEW

File: ci-operator/config/redhat-appstudio/image-controller/redhat-appstudio-image-controller-main.yaml [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+      memory: 200Mi
+tests:
+- as: image-controller-e2e
+  cluster_claim:
+    architecture: amd64
+    cloud: aws
+    owner: openshift-ci
+    product: ocp
+    timeout: 1h0m0s
+    version: "4.11"
```
Comment (psturc) [inline ci-operator/config/redhat-appstudio/image-controller/redhat-appstudio-image-controller-main.yaml:30]: should we update it to `4.12`?

I know there are other redhat-appstudio components that are targeting still 4.11, but it'd probably make sense to update them in a separate PR

Comment (tisutisu) [reply ci-operator/config/redhat-appstudio/image-controller/redhat-appstudio-image-controller-main.yaml]: Updated the version to 4.12 in the latest commit.

Comment (psturc) [reply ci-operator/config/redhat-appstudio/image-controller/redhat-appstudio-image-controller-main.yaml]: please update line 14 as well for using oc 4.12 client. otherwise lgtm

Comment (tisutisu) [reply ci-operator/config/redhat-appstudio/image-controller/redhat-appstudio-image-controller-main.yaml]: Updated line 14 to use oc 4.12 in the latest commit.

**Comment by tisutisu** [comment]
/pj-rehearse pull-ci-redhat-appstudio-image-controller-main-image-controller-e2e

**Review by psturc — APPROVED** [review]
/lgtm
